Transportation/Bike/BikeHireDockingStation/doc/spec.md

```diff
@@ -8,7 +8,7 @@ features and availability of bikes and free slots.
 ## Data Model
 
 
-A JSON Schema corresponding to this data model can be found [here](http://fiware.github.io/dataModels/Transportation/BikeHire/schema.json).
+A JSON Schema corresponding to this data model can be found [here](../schema.json).
```

Comment:
we want a link to the raw content not to the resource, so you need to keep the URL but pointing to the proper resource. thanks!